pkgs/development/python-modules/tensorrt/default.nix

```diff
@@ -33,20 +33,15 @@ buildPythonPackage rec {
 
   sourceRoot = ".";
 
-  runtimeDependencies = [
-    cudaPackages.tensorrt
-  ];
-
-  propagatedBuildInputs = [
-    cudatoolkit
+  buildInputs = [
+    cudaPackages.cudatoolkit
     cudaPackages.cudnn
     cudaPackages.tensorrt
```

Comment:
What's the difference between `cudaPackages.tensorrt` and the `tensorrt` in `pkgs/development/libraries/science/math/tensorrt/default.nix`?

Reply:
It should be the same package.